src/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Leaderboard-game</title>
+</head>
+<body>
+<header>
+        <h1 class="header">Leaderboard</h1>
+    </header>
+    <main>
+    <!-- section to add the name and score -->
+        <section class="add-scores">
+            <h2 class="scores-heading">Add your score</h2>
+            <form action="#" method="POST">
+                <div class="user-input">
+                    <label for="name">Name</label>
+                    <input type="text" id="name" placeholder="Enter Your name" required />
+                </div>
+                <div class="user-input">
+                    <label for="score">Score</label>
+                    <input type="number" id="score" placeholder="Update Your score" required />
+                </div>
+                <div class="submit-button">
+                    <button type="button" id="add-score">Submit</button>
+                </div>
+            </form>
+        </section> 
+  <section class="leaderboard">
+            <div class="header">
+                <h2 class="heading">Recent scores</h2>
+                <button id="scores-refresh" type="button">Refresh</button>
+            </div>
+            <ul class="scores-list">
+                <!-- example list elements (scores) -->
+                <li>
+                    <p class="country">India</p>
+                    <p class="score-value">100</p>
+                </li>
+                <li>
+                    <p class="country">United States</p>
+                    <p class="score-value">22</p>
+                </li>
+                <li>
+                    <p class="country">Australia</p>
+                    <p class="score-value">59</p>
+                </li>
+                <li>
+                    <p class="country">England</p>
+                    <p class="score-value">77</p>
+                </li>
+                <li>
+                    <p class="country">Germany</p>
+                    <p class="score-value">121</p>
+                </li>
+                <li>
+                    <p class="country">Japan</p>
+                    <p class="score-value">109</p>
+                </li>
+            </ul>
+        </section>   
+</main>
```

Comment:
- [ ] Awesome work my friend you did a great job by implementing the requirement of this milestone and even more. However, I would like to mention that it's very important that you always read the project requirements you're been given properly so you don't miss important details because attention to detail is a very important skill to have a developer.
   - [ ] In the project requirement it was mentioned that you stick to the layout of the wireframe provided and I can see you did everything well and thoroughly but missed the position of the layout a bit so kindly fix that. See below for more details:
   
   
<img width="1438" alt="Screenshot 2023-08-01 at 23 43 16" src="https://github.com/the-faizmohammad/leaderboard-game/assets/107192076/e9d0b276-5b21-48a2-926a-b259f48f4b4d">